modules/ocl-loader/src/main/java/org/eclipse/osc/modules/ocl/loader/Artifact.java

```diff
@@ -11,6 +11,7 @@ public class Artifact extends RuntimeBase {
 
     private String name;
     private String base;
+    private String type;
```

Comment:
If we know the list of Artifact types, we can make it an enum. Need not be a complete list. We can start with whatever values we know also.

Reply:
The idea here is to have "open" type. So it's not necessary "closed" values, it could be specific to some plugins.

Reply:
agreed.. IMHO, if we have an enum, then the user knows what values can be provided. The  enum can always be extended.